tests/test_flow_order_guardrail_config.py

```
@@ -0,0 +1,138 @@
+"""
+Configuration tests for the flow order guardrail.
+
+This module tests the allowlist and exclusion configuration for the
+flow order guardrail.
+"""
+
+from pathlib import Path
+from typing import List, Tuple
```

Ruff (0.14.14)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)